app/helpers/blob_helper.rb

```diff
@@ -8,11 +8,11 @@ def highlight(blob_name, blob_content, nowrap: false, continue: false)
     )
 
     begin
-      @lexer ||= Rugments::Lexer.guess(filename: blob_name, source: blob_content).new
+      @lexer ||= Rouge::Lexer.guess(filename: blob_name, source: blob_content)
```

Comment:
The test fails here because the constructor of `Lexer` calls `new(opts).lex(stream, &b)`, but `opts` is not actually used in the `lex` call. You can work around this issue by adding `.new` at the end of the line.